src/styles/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  padding: 32px 16px 16px;
+
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+
+  margin-bottom: 40px;
+
+  display: block;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+
+  margin-bottom: 16px;
+}
+
+.card__review-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  margin-bottom: 24px;
+}
+
+.card__review-value {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 16px;
+}
+
+.card__price-title {
+  color: #616070;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+.card__price-value {
+  color: #060b35;
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__buy-bottom {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+
+  transition: background-color 0.3s, color 0.3s;
+}
+
+.card__buy-bottom:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #f3f3f3;
+  border-color: #00acdc;
+}
```

Comment:
I'm still not sure about this one, because some students use for border color #00acdc, but you use here it two times..border: 1px solid #f3f3f3;
  border-color: #00acdc;
  I think you need only one and not in hover but just in .card__buy-bottom

Reply:
yes, I agree